src/utilities/formatter.js

```diff
@@ -0,0 +1,46 @@
+const prettier = require('prettier');
+const stripAnsi = require('strip-ansi');
+
+function formatter(unformattedCode, language) {
```

Comment:
export this on the first line, and add comments

Reply:
Done renaming the file and adding JSdoc.